index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
```

Comment:
- [ ] Good job @fredo509, but I suggest you add emoji's to your comments so that the student understands the tone you are trying to communicate 😎.
- [ ] Kingly, make sure you don't forget to highlight the effort of the student to encourage him, you could say something like `Good job for using...`,`Good job @fredo509...` ... 👍